src/js/mep-feature-speed.js

```diff
@@ -65,15 +66,19 @@
 					}
 				};
 
+				var speedLabel = function(speed) {
+					return t.options.speedLabel + ': Current speed ' + getSpeedNameFromValue(speed);
```

Comment:
Will adding strings here mess with the i18n.t that's already been done on `t.options.speedLabel` on line 14? it may be more consistent to do the i18n.t in this function